src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -40,153 +28,83 @@ const Button: React.FC<ButtonProps> = ({onPress, style, children}) => {
 };
 
 const CopyLink: React.FC<Omit<LinkProps, 'prompt'>> = ({username}) => {
-  const showCopyAlert = () =>
-    Alert.alert('🔗 Link Copied!', `${config.SHARE_URL}/u/${username}`, [{text: 'OK'}]);
-
-  const handleCopyLink = () => {
-    ShareUtils.copyToClipboard(username);
-
-    showCopyAlert();
-  };
   return (
     <View style={styles.copyLinkContainer}>
       <Text numberOfLines={1} ellipsizeMode="tail" style={styles.copyLinkUrl}>
         {`${config.SHARE_URL}/u/${username}`}
       </Text>
-
-      <Button onPress={handleCopyLink} styles={styles.copyLinkButton}>
-        <Text style={styles.copyLinkButtonLabel}>Copy Link</Text>
-      </Button>
     </View>
   );
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CopyLink` component seems to have lost its functionality of copying the link to the clipboard. This is a regression and should be fixed.
```diff
+ import Clipboard from 'react-native';
...
const CopyLink: React.FC<Omit<LinkProps, 'prompt'>> = ({username}) => {
+  const handleCopyLink = () => {
+    Clipboard.setString(`${config.SHARE_URL}/u/${username}`);
+  };
   return (
+    <TouchableOpacity onPress={handleCopyLink}>
       <View style={styles.copyLinkContainer}>
         <Text numberOfLines={1} ellipsizeMode="tail" style={styles.copyLinkUrl}>
           {`${config.SHARE_URL}/u/${username}`}
         </Text>
+     </TouchableOpacity>
       </View>
   );
};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->